devito/nodes.py

```diff
@@ -297,6 +297,10 @@ def ccode(self):
         """
         loop_body = [s.ccode for s in self.nodes]
 
+        # Disregard offsets for buffered dimensions
+        if self.dim.is_Buffered:
+            self.offsets = [0, 0]
```

Comment:
Should this be done here, or in the constructor? modifying object values in ``ccode`` is weird